block/aoe/aoe.go

```diff
@@ -211,3 +210,63 @@ func (s *Server) handleFrame(from net.Addr, iface *Interface, f *Frame) (int, er
 func (s *Server) Close() error {
 	return s.dev.Close()
 }
+
+// mustAssembleBPF assembles a BPF program to filter out packets not bound
+// for this server.
+func (s *Server) mustAssembleBPF(mtu int) []bpf.RawInstruction {
+	prog, err := bpf.Assemble([]bpf.Instruction{
+		// Load major address value from AoE header
+		bpf.LoadAbsolute{
+			Off:  16,
```

Comment:
i hope there is a const for location/size of the major field in header.  but seems like the aoe pkg does not expose it. :(